WordPress/build.gradle

```diff
@@ -186,7 +189,7 @@ android {
 
             // Limit string resources to Mag16 only for Jetpack
             // Note: this only affects included _locales_; variants for `values-night`, `values-land` and other configs are _still_ preserved in Jetpack and not filtered by this.
-            resourceConfigurations = ["ar", "de", "es", "fr", "he", "id", "it", "ja", "ko", "nl", "pt-rBR", "ru", "sv", "tr", "zh-rCN", "zh-rTW"]
+            resConfigs "ar", "de", "es", "fr", "he", "id", "it", "ja", "ko", "nl", "pt-rBR", "ru", "sv", "tr", "zh-rCN", "zh-rTW"
```

Comment:
According to [what I read in the documentation](https://developer.android.com/reference/tools/gradle-api/7.3/com/android/build/api/dsl/BaseFlavor#resConfigs(kotlin.collections.Collection)) and what Oguz told me recently [internal ref: pxLjZ-7b9-p2#comment-19303], `resConfig` is deprecated in favor of `resourceConfigurations` now (though sadly not all the guides in the Android docs have been updated)

Our current release scripts also [checks the `resourceConfigurations` field (and not the `resConfig` deprecated one)](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/build.gradle#L609-L617) at release time [to ensure our setup is consistent between here in `build.gradle` and in what locales are configured to be downloaded from GlotPress in our Fastlane scripts](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/fastlane/lanes/localization.rb#L425-L451), so it's another reason to use `resourceConfigurations` here and not `resConfigs`.